bug-reports.md

```diff
@@ -14,6 +14,26 @@ Once you've done a bit of searching and discovered there isn't prior art for you
 * What you expected to see.
 * What you actually saw.
 
+
+Following this template will help guarantee you are submitting a full bug report:
+
+**Environment:** affected software and versions (see below)
```

Comment:
Environment can go after the Summary and Steps to reproduce sections.

Also, it would be good to bring the existing environment content up in to your new section, instead of needing to reference later down the page.